.github/workflows/CI.yml

```diff
@@ -79,7 +79,22 @@ jobs:
       - name: Build and Run Tests
         run: |
           ./gradlew integTest yamlRestTest
-      - name: Multi Nodes Integration Testing
-        if: matrix.java == '21'
+  integMultiNodeTest:
+    needs: [spotless, javadoc]
+    strategy:
+      fail-fast: false
```

Comment:
Should we have this strategy only for `macos` and `windows`?

Reply:
When they stop being flaky we can remove this.  I don’t think it hurts to have it tho.